soh/src/overlays/actors/ovl_En_Part/z_en_part.c

```diff
@@ -299,11 +299,17 @@ void EnPart_Draw(Actor* thisx, PlayState* play) {
         gSPSegment(POLY_OPA_DISP++, 0x08, func_80ACEAC0(play->state.gfxCtx, 255, 255, 255, 180, 180, 180));
         gSPSegment(POLY_OPA_DISP++, 0x09, func_80ACEAC0(play->state.gfxCtx, 225, 205, 115, 25, 20, 0));
         gSPSegment(POLY_OPA_DISP++, 0x0A, func_80ACEAC0(play->state.gfxCtx, 225, 205, 115, 25, 20, 0));
-    } else if ((thisx->params == 9) && (strcmp((const char*)this->displayList, object_tite_DL_002FF0) == 0)) {
+        // #region SOH [Port] DL references are stored as resource paths so we need to use string comparison to check
+        // for equality. Additionally, we added a fallback for checking the parent actor ID to account for mods the
+        // override the Tektite skelton
+    } else if ((thisx->params == 9) && ((strcmp((const char*)this->displayList, object_tite_DL_002FF0) == 0) ||
+                                        (thisx->parent && thisx->parent->id == ACTOR_EN_TITE))) {
```

Comment:
Is `thisx->parent` enough here or does it need to compare to NULL? Genuinely don't know, figured I'd ask.

Reply:
Its enough as-is for a truthy check, but I'll push an explicit NULL check